net2/IntelTool.js

```diff
@@ -122,6 +122,11 @@ class IntelTool {
     intel.updateTime = `${new Date() / 1000}`
 
     await rclient.hmsetAsync(key, intel);
+    if (options && options.saveDomain) {
```

Comment:
separate intel:ip from intel:dns, don't mix them together. you can reuse the domain intel key from my PR.
https://github.com/firewalla/firewalla/pull/3435/files#diff-960c0bdfbb3d6071b1111659e23d3bfff91d7b946ec267d829e807ca7df86e65R86

the domain intel will be a cache of cloud intel, however the ip intel is a mixed intel from many information. so logically they are separate.